tests/src/main/java/com/axellience/vuegwt/tests/client/components/events/types/EmitTypesChildComponent.java

```diff
@@ -16,5 +17,12 @@ public void created() {
     vue().$emit("double", (double) 12);
     vue().$emit("float", (float) 12.5);
     vue().$emit("todo", new Todo("Hello World"));
+
+    vue().$emit("int", Js.asAny(10));
+    vue().$emit("integer", Js.asAny((Integer) 10));
+    vue().$emit("boolean", Js.asAny(false));
+    vue().$emit("double", Js.asAny(12));
+    vue().$emit("float", Js.asAny(12.5));
+    vue().$emit("todo", Js.asAny(new Todo("Hello World")));
```

Comment:
I'm not 100% sure that this is the right way to test this, but here's how to call the "new" override that uses Any. All of these still seem to work in this way, we just lean on `Js.asAny` instead of our own `@DoNotAutobox`.

Reply:
I'm not sure either, I'll merge to develop and try to see the test 👍